### PR TITLE
LPS-187449

### PR DESCRIPTION
**Pull request by marcosvictor22**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by marcosvictor22** [comment]
@david-gutierrez-mesa

**Comment by marcosvictor22** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/portalconfiguration/systemmanagement/VirtualInstances.testcase [hunk 1]
```diff
@@ -87,6 +87,49 @@ definition {
...
+			PagesAdmin.addPage(pageName = "Web Content Page");
+
+			Navigator.gotoSpecificURLPage(
+				pageName = "Web Content Page",
+				url = "http://www.able.com:8080");
+		}
+
+		task ("Then he can see how the content is performing on the page") {
+			ContentPerformance.openContentPerformance();
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/portalconfiguration/systemmanagement/VirtualInstances.testcase:130]: Missing the assertion step
Canonical URL is http://www.able.com:8080

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/portalconfiguration/systemmanagement/VirtualInstances.testcase [hunk 2]
```diff
@@ -87,6 +87,49 @@ definition {
...
+				userEmailAddress = "test@www.able.com",
+				virtualHostsURL = "http://www.able.com:8080");
+		}
+
+		task ("When the user creates a content page in the virtual instance") {
+			ProductMenu.gotoPortlet(
+				category = "Site Builder",
+				portlet = "Pages");
+
+			PagesAdmin.addPage(pageName = "Web Content Page");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/portalconfiguration/systemmanagement/VirtualInstances.testcase:121]: Create a content page then publish

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @marcosvictor22 , Could you refactor according to comments? Thanks

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase [hunk 3]
```diff
@@ -1499,4 +1499,48 @@ definition {
...
+	test CanonicalURLforContentPages {
+		property test.liferay.virtual.instance = "false";
+
+		task ("Given that the user creates a Virtual Instance") {
+			User.firstLoginUI();
+
+            SAML.addVirtualInstances(
+                mailDomain = "www.able.com",
+                virtualHost = "www.able.com",
+                webId = "www.able.com");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformance.testcase:1513]: Use headless macro to create virtual instance
```
	HeadlessPortalInstanceAPI.addPortalInstance(
		domain = "www.able.com",
		portalInstanceId = "www.able.com",
		virtualHost = "www.able.com");
```

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @marcosvictor22, please refactor according to comments. Thanks!

**Comment by ealonso** [comment]
Hey @marcosvictor22,

Please add the feedback added by @Tim-Cao and resend

**Pull request closed without merging**